src/NServiceBus.Transport.RabbitMQ.AcceptanceTests/NServiceBus.Transport.RabbitMQ.AcceptanceTests.csproj

```diff
@@ -11,12 +11,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="[15.*,16)" />
-    <PackageReference Include="NServiceBus" Version="[7.0.0-*,8)" />
-    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="[7.0.0-*,8)" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="NServiceBus" Version="7.0.0-beta0007" />
```

Comment:
Given that NServiceBus.AcceptanceTests.Sources will pull in the correct version, I don't think we need this package reference.

Reply:
True, I'll remove it.